code/game/objects/items/devices/scanners.dm

```diff
@@ -279,26 +287,25 @@ proc/healthanalyze(mob/living/M as mob, mob/living/user as mob, var/mode = 0)
 	else
 		message += "<span class='notice'><B>\icon [container] Results of [container] scan:</B><br></span>"
 	if(total_moles)
-		message += "[human_standard && abs(pressure - ONE_ATMOSPHERE) > 10 ? "<span class='bad'>" : "<span class='notice'>"] Pressure: [round(pressure,0.1)] kPa</span><br>"
-		var/o2_concentration = scanned.oxygen/total_moles
-		var/n2_concentration = scanned.nitrogen/total_moles
-		var/co2_concentration = scanned.carbon_dioxide/total_moles
-		var/plasma_concentration = scanned.toxins/total_moles
-
-		var/unknown_concentration =  1-(o2_concentration+n2_concentration+co2_concentration+plasma_concentration)
-
-		if(n2_concentration > 0.01)
-			message += "[human_standard && abs(n2_concentration - N2STANDARD) > 20 ? "<span class='bad'>" : "<span class='notice'>"] Nitrogen: [round(scanned.nitrogen, 0.1)] mol, [round(n2_concentration*100)]%</span><br>"
-		if(o2_concentration > 0.01)
-			message += "[human_standard && abs(o2_concentration - O2STANDARD) > 2 ? "<span class='bad'>" : "<span class='notice'>"] Oxygen: [round(scanned.oxygen, 0.1)] mol, [round(o2_concentration*100)]%</span><br>"
-		if(co2_concentration > 0.01)
-			message += "[human_standard ? "<span class='bad'>" : "<span class='notice'>"] CO2: [round(scanned.carbon_dioxide, 0.1)] mol, [round(co2_concentration*100)]%</span><br>"
-		if(plasma_concentration > 0.01)
-			message += "[human_standard ? "<span class='bad'>" : "<span class='notice'>"] Plasma: [round(scanned.toxins, 0.1)] mol, [round(plasma_concentration*100)]%</span><br>"
-		if(unknown_concentration > 0.01)
-			message += "<span class='notice'>Unknown: [round(unknown_concentration*100)]%<br></span>"
-
-		message += "[human_standard && !(scanned.temperature-T0C in range(0, 40)) ? "<span class='bad'>" : "<span class='notice'>"] Temperature: [round(scanned.temperature-T0C)]&deg;C"
+		message += "[human_standard && abs(pressure - ONE_ATMOSPHERE) > 10 ? "\red" : "\blue"] Pressure: [round(pressure,0.1)] kPa<br>"

```

Comment:
spans, also this entire section is copypasta and that needs to be fixed at some point